crypto_kem/kyber768/clean/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+cmake_minimum_required(VERSION 3.0)
+
+if(MSVC)
+    add_compile_options(/Wall /Wx)
+    # /Wall causes lots of errors, weaker options are /W1 or /W2 or /W3 or /W4
```

Comment:
/Wall causes lots of errors for kyber768, even /W1 catches a few things, so we'd have to do some more cleaning in the kyber768 code.